snowfakery/cci_mapping_files/declaration_parser.py

```diff
@@ -28,7 +28,7 @@ class AtomicDecl(T.NamedTuple):
 
     sf_object: str
     key: str
-    value: T.Union[T.List, str]
+    value: T.Union[T.List, str, date, int]
```

Comment:
If this is used for parsing, I don't think it will do what you want. In a Union, pydantic uses the first type that the input can successfully be cast to. I think it'd be a good idea to make the test_cli__explicit_file test assert that the values loaded from the file are in the expected format.

Reply:
(Oh, but it probably works because it's parsed by yaml first?)

Reply:
Sorry for the long delay. This is a NamedTuple not a Pydantic Model, so your concern does not arise.